### ISSUE TITLE
Training on COCO dataset

### ISSUE THREAD

**Issue by unlabeledData** [Stick-To/CenterNet-tensorflow#23]
Hi, thanks for your sharing your code!

1. Have you compared your results with the author's results on Pascal VOC?

2. Have you trained on COCO dataset? I trained Resnet-50 backbone on COCO dataset using your code, and I found that the `size` loss is extremely hard to converge.

3. Could you share your training loss curve and training hyper-parameters such as lr?

Thank you very much!

**Comment by Stick-To** [comment]
1 No
2 No
3 I have not keep them
I didn't pretrain my model, If you want to get better result, you could try official code

**Comment by unlabeledData** [comment]
> 1 No
> 2 No
> 3 I have not keep them
> I didn't pretrain my model, If you want to get better result, you could try official code

Thank you for your reply. 
So the mAP in your experiment on VOC with out pretraining model is?
